openerp/addons/base/ir/ir_model.py

```diff
@@ -1100,7 +1100,14 @@ def _update(self, model, module, values, xml_id=False, store=True, noupdate=Fals
     @api.model
     def ir_set(self, key, key2, name, models, value, replace=True, isobject=False, meta=None, xml_id=False):
```

Comment:
Another thing we could cleanup is the `ir_set` method itself, and the support for calling it via XML and YAML formats. I think his stuff is unused and officially deprecated since 9.0 as well. (see e.g. [here](https://github.com/odoo/odoo/blob/9.0/openerp/tools/convert.py#L462-L467))

This would remove a bit of dead code from `convert.py`, `yaml_import.py`, etc, and you'll be able to drop this method from `ir.model` as well. What do you think?